content/syllabuses/ncit-javascript.md

```diff
@@ -1,47 +1,25 @@
 ---
+_db_id: 37
 title: NCIT - JavaScript
 ---
 
-- {{% contentlink path="national-qualifications-framework/ncit/version-control" %}}
```

Comment:
We usually have them do these 3 first as they have just come out of the design thinking sprint and it and their research, presentations etc are fresh:
- {{% contentlink path="national-qualifications-framework/ncit/conducting-research-and-user-interviews" %}}
- {{% contentlink path="national-qualifications-framework/ncit/presenting-your-findings" %}}
- {{% contentlink path="national-qualifications-framework/ncit/write-a-report" %}}

But if this order makes more sense then I am happy

Reply:
I want to push to have the design sprint happen later. If it happens immediately as people join Umuzi then Make Umuzi Better is more like Make Random Assumptions About Umuzi. MRAAU just doesn't have the same ring to it.